homeassistant/components/zwave.py

```diff
@@ -505,6 +522,8 @@ def start_zwave(_service_or_event):
         hass.services.register(DOMAIN, SERVICE_TEST_NETWORK, test_network)
         hass.services.register(DOMAIN, SERVICE_STOP_NETWORK, stop_zwave)
         hass.services.register(DOMAIN, SERVICE_START_NETWORK, start_zwave)
+        hass.services.register(DOMAIN, SERVICE_RENAME_NODE, rename_node,
+                               descriptions[DOMAIN][SERVICE_RENAME_NODE])
```

Comment:
Please use voluptuous to validate the schema data

Examples:
 - [define schema](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/__init__.py#L82-L85)
 - [pass in during registration](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/climate/__init__.py#L271-L274)